src/tokens/core.js

```diff
@@ -37,57 +37,57 @@ export default {
     TransparentScrimDarker: "rgba(0, 0, 0, 0.50)",
   },
   BoxShadow: {
-    Low: ' 0px 1px 3px rgba(87, 102, 117, 0.2)',
-    Medium: ' 0px 3px 8px rgba(87, 102, 117, 0.15)',
-    High: ' 0px 10px 20px rgba(87, 102, 117, 0.16)',
-    Top: ' 0px -1px 3px rgba(87, 102, 117, 0.2)',
-    Left: ' -1px 0px 3px rgba(87, 102, 117, 0.2)',
-    Right: ' 1px 0px 3px rgba(87, 102, 117, 0.2)',
-    Focus: '0 0 0 2px rgba(82, 138, 224, 0.8)',
```

Comment:
It doesn't look like we'll be able to use these definitions in react native so these can be put back.